tools/travis-upload-to-gdrive.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Can we add a comment block.
What this script for, possible arguments and used env variables.